### PR TITLE
Distributions Gallery: Add ExGaussian

### PR DESCRIPTION
**Pull request by aleicazatti**
## Notes:

I tried to derive the PDF and CDF from the equations on the [Wikipedia page](https://en.wikipedia.org/wiki/Exponentially_modified_Gaussian_distribution) using $\nu$, but in  the docstring of the distribution, the PDF expression is a little different.

### CODE REVIEW

**Comment by aloctavodia** [comment]
Are the differences indicate a potential mistake somewhere or just two ways to express the same? Did you compare the mathematical expressions for the pdf and cdf with the code for them? Are

**Comment by aleicazatti** [comment]
I was checking. It seems they are equivalent. I dont' know if we should use the docstring/code expressions or the ones in Wikipedia.

**Comment by aloctavodia** [comment]
Not sure, probably match the code.

File: docs/examples/gallery/exgaussian.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+---
+tags: [remove-input]
+mystnb:
+  image:
+    alt: Ex-Gaussian Distribution PDF
+---
+
+import arviz as az
+from preliz import ExGaussian
+az.style.use('arviz-doc')
```
Comment (aloctavodia) [inline docs/examples/gallery/exgaussian.md:29]:
```suggestion
from preliz import ExGaussian, style
style.use('preliz-doc')
```

**Pull request merged**